src/lazy/binary/raw/v1_1/e_expression.rs

```diff
@@ -79,26 +56,66 @@ impl<'top> BinaryEExpression_1_1<'top> {
         macro_ref: MacroRef<'top>,
         bitmap_bits: u64,
         input: BinaryBuffer<'top>,
+        length_offset: u8,
         bitmap_offset: u8,
         args_offset: u8,
     ) -> Self {
         Self {
             bitmap_bits,
             input,
             macro_ref,
+            length_offset,
             bitmap_offset,
             args_offset,
             cache: None,
         }
     }
 
-    pub fn with_arg_expr_cache(
+    pub(crate) fn with_arg_expr_cache(
         mut self,
         cache: &'top [ValueExpr<'top, BinaryEncoding_1_1>],
     ) -> Self {
         self.cache = Some(cache);
         self
     }
+
+    /// Returns a span of bytes representing the opcode and macro address.
+    /// Depending on the encoding, these may be distinct (for example, the span: `0xF4 0x01`,
+    /// where the `0xF4` is the opcode and the `0x01` is the `FlexUInt` address) or combined
+    /// (for example: `0x00` is both an opcode and a macro address).
+    pub fn opcode_and_address_span(&self) -> Span<'top> {
+        self.input.slice(0, self.length_offset as usize).into()
+    }
+
+    /// Returns `true` if this binary e-expression includes a length prefix.
+    pub fn has_length_prefix(&self) -> bool {
+        // If these offsets are equal, there are no bytes representing the length.
+        self.length_offset == self.bitmap_offset
```

Comment:
```suggestion
        // If these offsets are equal, there are no bytes representing the length.
        self.length_offset != self.bitmap_offset
```
If I'm reading this correctly. Though this function does appear unused.